### ISSUE TITLE
Layout subplot labelling documentation and formatting

### ISSUE THREAD

**Issue by mangecoeur** [pyviz/holoviews#3934]
Layout adds subplot labels 'A, B, C' etc next to each plot, but there is no information on how these are assigned or how to modify them e.g. to set the size and positioning, or to change the labels completely (such as to use lowercase letters).

In my they are appearing in a very large font with a lot of padding and do not seem to be affected by font size settings.

I'm using the matplotlib backend

**Comment by jbednar** [comment]
You can control those with the options sublabel_format, sublabel_position, and sublabel_size. I seem to remember there being a docs page about that, but don't see it in a quick search just now, so maybe we deleted it?  Please do a more thorough search for those terms and if they are documented, please link to it here, and if they are not, please let us know so that we can restore those docs.

**Comment by mangecoeur** [comment]
Thanks for the answer - helps a lot knowing what keywords to look for.

As far as I can see there is no dedicated page or section in the documentation that comes in in a regular google search. Searching within the holoviews docs (http://holoviews.org/search.html?q=sublabel_format ) finds code snippets with the keywords but no tutorial or API docs.

A gitter discussion also popped up https://gitter.im/ioam/holoviews/archives/2015/06/25 , from that I realised I should try hv.help (I had been looking at the function documentation in .opts which sometimes has that same information) and that does show some information on the keywords. 

So it looks like if there was a section on this it got removed from the main docs.

**Comment by jbednar** [comment]
I don't think it was a full page, but it was a section in another page, e.g. one about customization.  Probably has been missing for some time!  I *think* @jlstevens wrote that, so I'll assign this to him.

**Comment by DancingQuanta** [comment]
I think this is where the usage of subfigure labels are described. https://github.com/holoviz/holoviews/blob/63cfac9e104b134f9dcb7eb0f1b16196df16e2db/doc/Tutorials/Containers.ipynb
The tutorials are acceesed from this page http://holoviews.org/Tutorials/, but where is the tutorials on the homepage? I have to google the to find the tutorial page.
